test/libsolidity/semanticTests/array/copying/array_copy_storage_storage_static_static.sol

```diff
@@ -13,5 +13,7 @@ contract c {
     }
```

Comment:
I think we don't have a syntax test for the opposite case (i.e. `data2` longer than `data1` which should result in a compilation error). A least I could not find one. Could you add it?

There's also no test for assigning a dynamic array to a static one. Especially the case when the dynamic one is longer. We don't have a test for assigning longer dynamic one to a shorter dynamic one either.

Reply:
I'll add such tests, so we confirm which conversions are available and to have them covered in syntax/semantic tests.